test/integration/http.go

```diff
@@ -59,6 +59,11 @@ func (r *http) makeRequests() error {
 	funcs := make(map[string]func() status)
 	for _, src := range testPods {
 		for _, dst := range testPods {
+			if src == "t" && dst == "t" {
```

Comment:
It tests that istio does not mess up things that don't have proxy. This might be obvious but since we do more and more invasive things (like cluster-wide), it's good to verify that.